### DISCUSSION TITLE
Ficheiro SAF-T Rejeitados

### DISCUSSION THREAD

**Discussion by inmx** [assoft-portugal/SAF-T-AO#169] [General]
Saudação pessoal, alguns clientes nossos têm os seus ficheiros SAF-T rejeitados por parte do validador da AGT, deste modo gostaria de saber quais são os motivos que estão na base da rejeição do ficheiro SAF-T visto que se encontram nos conformes?

Alguém poderia me ajudar

**Comment by cryptolopes** [comment]
https://github.com/assoft-portugal/SAF-T-AO/discussions/208
https://github.com/assoft-portugal/SAF-T-AO/discussions/167
